### PR TITLE
unifi6: 6.5.53 -> 6.5.54, nixos/unifi: Apply log4j2 mitigation

### PR DESCRIPTION
**Pull request by zhaofengli**
###### Motivation for this change

This mitigates #150288 for `unifi6` and `unifi5`. I have only tested `unifi6` in my setup.

Summary:

- `unifiLTS`: `log4j-1.2.17.jar` (not vulnerable)
- `unifi5`: `log4j-core-2.12.1.jar` (mitigated with `log4j2.formatMsgNoLookups` switch)
- `unifi6`: `log4j-core-2.13.3.jar` -> `log4j-core-2.15.0.jar`

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by pennae — APPROVED** [review]
can't test unifi5 either, but unifi6 works and module changes LGTM.

File: nixos/modules/services/networking/unifi.nix [hunk 1]
```diff
@@ -7,6 +7,7 @@ let
     @${cfg.jrePackage}/bin/java java \
         ${optionalString (cfg.initialJavaHeapSize != null) "-Xms${(toString cfg.initialJavaHeapSize)}m"} \
         ${optionalString (cfg.maximumJavaHeapSize != null) "-Xmx${(toString cfg.maximumJavaHeapSize)}m"} \
+        -Dlog4j2.formatMsgNoLookups=true \
```
Comment (pennae) [inline nixos/modules/services/networking/unifi.nix:10]: one optional nit though: set this parameter only if the unifi package is known to be vulnerable (eg versionBefore "6.5.54")

**Review by pennae — APPROVED** [review]

**Review by pennae — APPROVED** [review]
quicker than ofborg, lol

**Review by Patryk27 — APPROVED** [review]
🚀

**Review by mweinelt — APPROVED** [review]

**Pull request merged**